packages/features/ee/workflows/components/WorkflowDetailsPage.tsx

```
@@ -33,13 +41,31 @@ interface Props {
+    workflowId,
+    selectedOptions,
+    setSelectedOptions,
+    teamId,
+    isOrg,
+    allOptions,
+    permissions: _permissions,
+  } = props;
   const { t } = useLocale();
   const router = useRouter();
 
+  const permissions = permissions || {
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)